app/models/purchase.rb

```diff
@@ -87,7 +87,7 @@ def replace_increase!(new_purchase_params)
   end
 
   def self.csv_export_headers
-    ["Purchases from", "Storage Location", "Quantity of Items", "Variety of Items", "Amount spent in Cents"]
+    ["Purchases from", "Storage Location", "Quantity of Items", "Variety of Items", "Amount spent in Cents", "Purchased Date"]
```

Comment:
Do you think a person reading the CSV may want to see the "Purchase Date" in an earlier column? I imagine that we want the most critical data points for finding in earlier columns. What do you think?

Reply:
Great point! I will move the "Purchase Date" to the column after "Storage Location"